### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by lewinkedrs**
### What does this PR do?
Adds documentation front-end for the repository.

### Motivation

Documentation pages allow us to add more context than what we can include in a readme. They also allow for a better customer experience to discover and learn about the accelerator.

### More

- [ X ] Yes, I have tested the PR using my local account setup  (Provide any test evidence report under Additional Notes)
- [ ] Yes, I have added a new example under [examples](https://github.com/aws-observability/terraform-aws-eks-blueprints/tree/main/examples) to support my PR
- [ ] Yes, I have created another PR for add-ons under [add-ons](https://github.com/aws-samples/eks-blueprints-add-ons) repo (if applicable)
- [ ] Yes, I have updated the [docs](https://github.com/aws-observability/terraform-aws-eks-blueprints/tree/main/docs) for this feature
- [ X ] Yes, I ran `pre-commit run -a` with this PR

**Note**: Not all the PRs required examples and docs except a new pattern or add-on added.

### For Moderators
- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

**Comment by bonclay7** [comment]
TODO
- [x] simplify content structure
- [ ] revise content and examples
- [ ] min iam permissions
- [ ] fix links

File: docs/index.md [hunk 1]
```diff
@@ -1,19 +1,56 @@
 # AWS Observability Accelerator for Terraform
 
-![GitHub](https://img.shields.io/github/license/aws-observability/terraform-aws-observability-accelerator)
+Welcome to the AWS Observability Accelerator for Terraform!
 
-Welcome to AWS Observability Accelerator for Terraform!
+The AWS Observability accelerator is a set of Terraform modules to help you
+configure Observability for your workloads and environemnts  with AWS
+Observability services. This project proposes a core module to bootstrap
+your cluster with the AWS Distro for OpenTelemetry (ADOT) Operator for EKS,
```
Comment (lewinkedrs) [inline docs/index.md:8]: "your cluster with"

Should we say something like "your environment with"  ? In the future we may want to expand beyond EKS/ECS clusters.

File: docs/index.md [hunk 2]
```diff
@@ -1,19 +1,56 @@
...
+Amazon Managed Service for Prometheus, Amazon Managed Grafana.
+Additionally we have a set of workloads modules to leverage curated ADOT
```
Comment (lewinkedrs) [inline docs/index.md:10]: "workloads" to "workload" also need to do this in the nav bar.

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+  palette:
+    primary: indigo
+    accent: grey
+
+nav:
+  - Home: index.md
+  - Concepts: concepts.md
+  - Amazon EKS cluster monitoring: eks.md
+  - Workloads monitoring:
```
Comment (lewinkedrs) [inline mkdocs.yml:29]: These should all be capitalized like "Workload Monitoring" since they are titles.

**Comment by bonclay7** [comment]
@lewinkedrs let's merge this and switch links to public after the release

**Review by bonclay7 — DISMISSED** [review]

**Review by bonclay7 — APPROVED** [review]
LGTM, but I committed as well :D

**Pull request merged**